src/asim/configs/resident/vehicle_type_choice_op4.csv

```diff
@@ -127,82 +127,26 @@ util_age18,Age 18 ASC,(age==18),coef_age18
-util_msp_ev,Minneapolis - Electric,"@(CBSA == 'MSP') & ((df.fuel_type == 'PEV') | (df.fuel_type == 'BEV'))",coef_msp_ev
-util_msp_age,Minneapolis - Age,"@(CBSA == 'MSP') * df.age",coef_msp_age
-util_dca_van,Washington DC - Van,"@(CBSA == 'DCA') & (df.body_type.str.contains('Van'))",coef_dca_van
-util_dca_suv,Washington DC - SUV,"@(CBSA == 'DCA') & (df.body_type.str.contains('SUV'))",coef_dca_suv
-util_dca_pu,Washington DC - Pickup,"@(CBSA == 'DCA') & (df.body_type.str.contains('Pickup'))",coef_dca_pu
-util_dca_mc,Washington DC - Motorcycle,"@(CBSA == 'DCA') & (df.body_type.str.contains('Motorcycle'))",coef_dca_mc
-util_dca_hyb,Washington DC - Hybrid,"@(CBSA == 'DCA') & (df.fuel_type == 'Hybrid')",coef_dca_hyb
-util_dca_ev,Washington DC - Electric,"@(CBSA == 'DCA') & ((df.fuel_type == 'PEV') | (df.fuel_type == 'BEV'))",coef_dca_ev
-util_dca_age,Washington DC - Age,"@(CBSA == 'DCA') * df.age",coef_dca_age
-util_oreg_van,Oregon - Van,"@(CBSA == 'OREG') & (df.body_type.str.contains('Van'))",coef_oreg_van
-util_oreg_suv,Oregon - SUV,"@(CBSA == 'OREG') & (df.body_type.str.contains('SUV'))",coef_oreg_suv
-util_oreg_pu,Oregon - Pickup,"@(CBSA == 'OREG') & (df.body_type.str.contains('Pickup'))",coef_oreg_pu
-util_oreg_mc,Oregon - Motorcycle,"@(CBSA == 'OREG') & (df.body_type.str.contains('Motorcycle'))",coef_oreg_mc
-util_oreg_hyb,Oregon - Hybrid,"@(CBSA == 'OREG') & (df.fuel_type == 'Hybrid')",coef_oreg_hyb
-util_oreg_ev,Oregon - Electric,"@(CBSA == 'OREG') & ((df.fuel_type == 'PEV') | (df.fuel_type == 'BEV'))",coef_oreg_ev
-util_oreg_age,Oregon - Age,"@(CBSA == 'OREG') * df.age",coef_oreg_age
-util_ohio_van,Ohio - Van,"@(CBSA == 'OHIO') & (df.body_type.str.contains('Van'))",coef_ohio_van
-util_ohio_suv,Ohio - SUV,"@(CBSA == 'OHIO') & (df.body_type.str.contains('SUV'))",coef_ohio_suv
-util_ohio_pu,Ohio - Pickup,"@(CBSA == 'OHIO') & (df.body_type.str.contains('Pickup'))",coef_ohio_pu
-util_ohio_mc,Ohio - Motorcycle,"@(CBSA == 'OHIO') & (df.body_type.str.contains('Motorcycle'))",coef_ohio_mc
-util_ohio_hyb,Ohio - Hybrid,"@(CBSA == 'OHIO') & (df.fuel_type == 'Hybrid')",coef_ohio_hyb
-util_ohio_ev,Ohio - Electric,"@(CBSA == 'OHIO') & ((df.fuel_type == 'PEV') | (df.fuel_type == 'BEV'))",coef_ohio_ev
-util_ohio_age,Ohio - Age,"@(CBSA == 'OHIO') * df.age",coef_ohio_age
+util_san_van,San Diego - Van,"@(CBSA == 'SAN') & (df.body_type_num_coded==5)",coef_san_van
+util_san_suv,San Diego - SUV,"@(CBSA == 'SAN') & (df.body_type_num_coded==4)",coef_san_suv
+util_san_pu,San Diego - Pickup,"@(CBSA == 'SAN') & (df.body_type_num_coded==3)",coef_san_pu
+util_san_mc,San Diego - Motorcycle,"@(CBSA == 'SAN') & (df.body_type_num_coded==2)",coef_san_mc
+util_san_hyb,San Diego - Hybrid,"@(CBSA == 'SAN') & (df.fuel_type_num_coded==4)",coef_san_hyb
+util_san_ev,San Diego - Electric,"@(CBSA == 'SAN') & ((df.fuel_type_num_coded==5) | (df.fuel_type_num_coded==1))",coef_san_ev
+util_san_age,San Diego - Age,"@(CBSA == 'SAN') * df.age",coef_san_age
```

Comment:
suggest we add a preprocessor variable for this SAN string comparison now so that sharrow will run faster in the future.

Reply:
Done. Though since they're all True (that's simply how I defined it in the preprocessor) would we be able to get away with just removing it from the expressions?